src/launcher/index.ts

```diff
@@ -154,9 +164,22 @@ async function main() {
       flowrWindow = await createFlowrWindow(store)
       FullScreenManager.applyDefaultActionOnWindow(flowrWindow)
       applicationManager.flowrWindow = flowrWindow
+
       flowrWindow.on('close', () => {
         flowrWindow = null
       })
+
+      flowrWindow.webContents.on('new-window', async (event, url) => {
+        event.preventDefault()
+
+        await openBrowserWindow(store, event, {
+          clearBrowsingDataAtClose: false,
+          openUrl: url,
+          maxTab : 1,
+          enableVirtualKeyboard: true,
```

Comment:
this property is set from the store: https://github.com/taktik/flowr-desktop/pull/33/files#diff-47ee3c25cc8e4af787f0e42cbbd44f6f34fc8a5986419eee8efc5b3adcaef37dR69